shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/framework/container/compose/DockerComposedContainer.java

```diff
@@ -33,7 +34,7 @@ public final class DockerComposedContainer extends BaseComposedContainer {
     
     private final DatabaseType databaseType;
     
-    private final ShardingSphereProxyDockerContainer proxyContainer;
+    private final ShardingSphereProxyClusterContainer proxyContainer;
```

Comment:
Could we remove `ShardingSphereProxyDockerContainer` in this PR?

Reply:
sure thing. I intended to remove it in next PR :)

Reply:
> sure thing. I intended to remove it in next PR :)

It's better to remove unused files or codes in time.

Reply:
> > sure thing. I intended to remove it in next PR :)
> 
> It's better to remove unused files or codes in time.

yeah, you are right.